bigquery_etl/shredder/delete.py

```diff
@@ -203,28 +203,59 @@ def delete_from_partition(
         job_config.write_disposition = bigquery.WriteDisposition.WRITE_TRUNCATE
 
     def create_job(client):
-        field_condition = " OR ".join(
-            f"""
-             {field} IN (
-               SELECT
-                 {source.field}
-               FROM
-                 `{sql_table_id(source)}`
-               WHERE
-            """
-            + " AND ".join((source_condition, *source.conditions))
-            + ")"
-            for field, source in zip(target.fields, sources)
-        )
-        query = reformat(
-            f"""
-            {"DELETE" if use_dml else "SELECT * FROM"}
-              `{sql_table_id(target)}`
-            WHERE
-              ({field_condition}){"" if use_dml else " IS NOT TRUE"}
-              AND {partition.condition}
-            """
-        )
+        if use_dml:
+            field_condition = " OR ".join(
+                f"""
+                 {field} IN (
+                   SELECT
+                     {source.field}
+                   FROM
+                     `{sql_table_id(source)}`
+                   WHERE
+                """
```

Comment:
Indentation is a bit off (I see it was that way before, but since we're changing this we may as well fix it).
```suggestion
                f"""
                {field} IN (
                  SELECT
                    {source.field}
                  FROM
                    `{sql_table_id(source)}`
                  WHERE
                """
```

Reply:
Ditto for the similar query below.